product-backend/src/main/java/org/kainos/ea/DropwizardWebServiceApplication.java

```diff
@@ -10,6 +10,9 @@
 import org.kainos.ea.resources.AuthController;
 
 import org.kainos.ea.resources.JobRoleController;
+import org.kainos.ea.resources.JobSpecificationController;
+
+
 
 
 public class DropwizardWebServiceApplication extends Application<DropwizardWebServiceConfiguration> {
```

Comment:
you don't need 4 lines separation between import and class - 1 line is a standard